uw_sws/section.py

```diff
@@ -411,8 +411,17 @@ def _json_to_section(section_data,
             attribute = "meets_%s" % day_data["Name"].lower()
             setattr(meeting, attribute, True)
 
-        meeting.start_time = meeting_data["StartTime"]
-        meeting.end_time = meeting_data["EndTime"]
+        if len(meeting_data["StartTime"]) and\
```

Comment:
Can you use parens for these nested blocks? The whitespace looks funny.

Reply:
If use parens,
uw_sws/section.py:415:13: E129 visually indented line with same indent as next logical line
uw_sws/section.py:421:13: E129 visually indented line with same indent as next logical line